Reddit/Memes/Scrape.py

```diff
@@ -1,47 +1,87 @@
 import requests
-import random
-import urllib.request
+import os
 import uuid
 import re
-import os
+from concurrent.futures import ThreadPoolExecutor
+from bs4 import BeautifulSoup
+
+# Function to fetch free proxies from https://free-proxy-list.net/
+def get_free_proxies():
+    response = requests.get("https://free-proxy-list.net/")
+    soup = BeautifulSoup(response.content, "html.parser")
```

Comment:
Instead of using Beautiful soup, It would be great if you could filter the proxies using regex. I checked the source code for the free-proxy-list. You could extract the proxies 
![image](https://user-images.githubusercontent.com/29656761/235337741-57041341-e23c-4cdf-b318-1e323a64813d.png)